### DISCUSSION TITLE
[Feature] Remove location metadata from asset

### DISCUSSION THREAD

**Discussion by ayykamp** [immich-app/immich#10005] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometimes you accidentally add a location to an asset. It would be nice to have an easy option to remove it again. Thanks

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7950
